### PR TITLE
Support for due date

### PR DESCRIPTION
**Pull request by ys**
This will allow you to use `due dates` for cards.

How does it look in a `onboarding.markdown` file?

```
# List
## Card to be done by the next day of your hiring +1d
## Card to be done 10 days before your hiring date -10d
```

### CODE REVIEW

File: assets/javascripts/app.coffee [hunk 1]
```diff
@@ -301,18 +301,29 @@ fillBoard = (trelloBoard, lists) ->
...
+    if expectedDateMatch[1] == "+"
+      today.setDate(today.getDate() + parseInt(expectedDateMatch[2]))
+    if expectedDateMatch[1] == "-"
+      today.setDate(today.getDate() - parseInt(expectedDateMatch[2]))
+    today
+
 createCards = (list) ->
   _.map(list.cards, (card, index) ->
-    name = card.name.replace(/\[(.*)\]/, '')
+    name = card.name.replace(/\[(.*)\]/, '').replace(/(\+|-)\d+d/, '')
```
Comment (idan) [inline assets/javascripts/app.coffee]: Is this about chopping off the +/- date bit from the card title?

Comment (ys) [inline assets/javascripts/app.coffee]: yes, so that it does not show in the card name

**Comment by idan** [comment]
👍

**Comment by jroes** [comment]
Hey @ys should probably take what you wrote in the description of this PR and add it to the docs somewhere.

**Comment by ys** [comment]
It is on our internal docs repository

**Pull request merged**